### ISSUE TITLE
i am using vscode in windows 11 latest version

### ISSUE THREAD

**Issue by Rohit8481** [microsoft/vscode#134824]
Issue Type: <b>Performance Issue</b>

i am using vscode in windows 11 latest version and it is lagging ans frezing even to scroll a single line it is strugguling dont know what to do whole work is stucked.

VS Code version: Code 1.61.0 (ee8c7def80afc00dd6e593ef12f37756d8f504ea, 2021-10-07T18:13:09.652Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz (8 x 1190)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.79GB (2.67GB free)|
|Process Argv|--crash-reporter-id 61f28748-45dc-450e-9fb7-01b387b1c7a4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	    84	  6920	code main
    0	   101	 10392	   shared-process
    0	    66	 13136	     ptyHost
    0	    37	 11120	   utility
    0	    79	 11364	   window (Issue Reporter)
    4	   181	 11916	   window (index.html - techcorner - Visual Studio Code)
    2	   157	  2112	     extensionHost
    0	   125	 10064	       "C:\Users\Rohit Pandit\AppData\Local\Programs\Microsoft VS Code\Code.exe" "c:\Users\Rohit Pandit\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\html-language-features\server\dist\node\htmlServerMain" --node-ipc --clientProcessId=2112
    0	    69	  8328	     watcherService
    0	    25	 12140	   crashpad-handler
    0	   160	 13080	   gpu-process
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (index.html - techcorner - Visual Studio Code)
|    Folder (techcorner): 9 files
|      File types: css(2) js(2) cfg(1) html(1)
|      Conf files:;
```

</details>
<details><summary>Extensions (21)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.6.1
code-runner|for|0.11.6
vscode-pull-request-github|Git|0.31.0
beautify|Hoo|1.5.0
firebase-explorer|jsa|0.3.3
python|ms-|2021.10.1317843341
vscode-pylance|ms-|2021.10.0
jupyter|ms-|2021.9.1001312534
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
remote-wsl|ms-|0.58.2
material-icon-theme|PKi|4.10.0
java|red|0.82.0
LiveServer|rit|5.6.1
vscodeintellicode|Vis|1.2.14
vscode-java-debug|vsc|0.36.0
vscode-java-dependency|vsc|0.18.8
vscode-java-pack|vsc|0.18.5
vscode-java-test|vsc|0.32.0
vscode-maven|vsc|0.34.0
vscode-icons|vsc|11.6.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30358481
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwt:30378364
pythonvssor306:30344512
bridge0708:30335490
pygetstartedt2:30371810
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30364666
pythonvspyt187:30373474
pydsgst2:30361792
vsqsis200:30374795
vsaa593:30376534
vssld246cf:30379912

```

</details>

**Comment by deepak1556** [comment]
Does the issue happen when you launch with `code --disable-renderer-accessibility` ?

**Comment by Rohit8481** [comment]
> Does the issue happen when you launch with `code --disable-renderer-accessibility` ?

yes i also downfraded it to 1.45 something but still not fixed. My whole work is stucked dont know what to do i have to change my ide now.

**Comment by Xoben** [comment]
> Does the issue happen when you launch with `code --disable-renderer-accessibility` ?

Yes, I am currently running 1.61.1 bunch of extensions just incase
